### ISSUE TITLE
MotionEyeOS records only when the page is open in the browser

### ISSUE THREAD

**Issue by orebla85** [ccrisan/motioneyeos#3050]
### motionEyeOS Version

I am running motionEyeOS version:

motionEye Version | 0.42.1
Motion Version | 4.2.2+gitUNKNOWN
OS Version | motionEyeOS 20200606

### Board Model

I am using the following board/model: Raspberry PI 3+

### Camera
I have 2 cameras: Reolink and YiHome with RSTP url to MotioneyeOS

### Network Connection

My motionEyeOS unit is connected to the network via: Ethernet

### Log Files

I consider the following log files relevant to this issue:
[motioneye(2).log](https://github.com/user-attachments/files/18155999/motioneye.2.log)

It's a very strange problem. I set the MotionEye to record continuously. Unfortunately it only starts recording when I am connected via the web interface. If I exit the browser it stops recording. But if I stay on the page it records correctly and saves the video. Have I made any settings wrong?
Thank you

**Comment by Marijn0** [comment]
Have you enabled motion detection? This should be activated even if you are using continuous recording. 
![Screenshot](https://github.com/user-attachments/assets/22ca7755-53e6-47c0-beb4-1f51ffc702e7)

You can also try setting 'emulate_motion on' in the extra motion options.

**Comment by orebla85** [comment]
HI. No, I turned off motion detection. I didn't think it was necessary with continuous recording. I'll activate it now and see what happens. Thank you

**Comment by orebla85** [comment]
No, it doesn't work. not even activating motion detection. If I set the recording mode in 'motion triggered' it works perfectly. But if I put 'continuous recording' it doesn't record anything.

**Comment by Marijn0** [comment]
Maybe your video file is getting too large. 
Try setting the 'Maximum Movie Length' to limit the file size.

Could you please share your log file again?
